Makefile.work

```diff
@@ -181,6 +182,7 @@ endif
 DOCKER_LOCKFILE_SAVE := $(DOCKER_LOCKDIR)/docker_save.lock
 $(shell mkdir -m 0777 -p $(DOCKER_LOCKDIR))
 $(shell [ -f $(DOCKER_LOCKFILE_SAVE) ] || (touch $(DOCKER_LOCKFILE_SAVE) && chmod 0777 $(DOCKER_LOCKFILE_SAVE)))
+$(shell sudo rm -rf $(DOCKER_ROOT) && mkdir -p $(DOCKER_ROOT))
```

Comment:
is there way to cleanup this folder?

Reply:
Yes, the folder can be cleaned up when calling "make reset".
https://github.com/Azure/sonic-buildimage/blob/356b51f4d6c903e5a40c9a3ab01feae5182e0724/Makefile.work#L466

For azp, we will do cleanup for the whole workspace.

Reply:
We have already added cleanup for "fsroot*", it can be cleaned up by "make reset".